.github/workflows/build_and_test.yaml

```diff
@@ -42,7 +49,7 @@ jobs:
         # NOTE: macos-10.15 is required for now, to work around issues with our
         # build system.  See related comments in
         # .github/workflows/custom-actions/build-packager/action.yaml
-        os: ["ubuntu-latest", "macos-10.15", "windows-latest", "self-hosted-linux-arm64"]
+        os: ["ubuntu-latest", "macos-10.15", "windows-2019", "self-hosted-linux-arm64"]
```

Comment:
Much like how you have a comment for why we use `macos-10.15`, you should have a comment for why we use `windows-2019`.

Reply:
Good point.  Done.